### PR TITLE
Added the very beginnings to a Seattle guide

### PR DESCRIPTION
**Pull request by alukach**
Added some very basic stuff to the Seattle WA guide.  I made this pull request by hitting the 'Compare & pull request' button that is added when Github notices that you recently made a commit to a branch.

Pull Request Button

### CODE REVIEW

**Comment by bbaughn** [comment]
I think there's a typo and you meant "Best Pub".  Let me try correcting that and pushing it here.

**Comment by alukach** [comment]
Yeah, I saw that.  You can either fix it, or we could use it as a place to
make a proper 'Bug Report'.

File: washington/seattle.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Seattle WA
+
+Home of the Space Needle: America's favorite place to do Space Heroin!
+
+## Restaurants
+
+### Best Put
```
Comment (bbaughn) [inline washington/seattle.md]: This should probably be "Pub" ya dummy

**Comment by alukach** [comment]
So, it appears that you pulled this feature branch, made a change, and then pushed it back up to the remote server.  It now automatically came up in this Pull Request thread.

So, it's like "Not ready yet... make a fix... now is it ready?"  all in the same Pull Request, no point of making multiple pull requests on the same branch.

**Comment by bbaughn** [comment]
Yep

**Comment by alukach** [comment]
So, it looks like you can close/re-open Pull Requests with the button at the bottom of the chat box.

screen shot 2013-07-17 at 4 08 43 pm

**Pull request merged**